TidepoolOnboardingKitUI/TidepoolOnboardingUI.swift

```diff
@@ -29,11 +29,18 @@ public final class TidepoolOnboardingUI: ObservableObject, OnboardingUI {
 
     var therapySettings: TherapySettings? {
         didSet {
-            guard therapySettings != oldValue, let therapySettings = therapySettings else { return }
+            guard let therapySettings = therapySettings else { return }
```

Comment:
[nit] what should happen in the `nil` case, especially if it becomes `nil` from actual settings?

Reply:
Same as above. Will change to precondition.